docs/design/router.md

```diff
@@ -138,18 +138,18 @@ Allows defining arbitrary routing rules using a JavaScript function. The functio
 
 > JavaScript-based functionality is disabled by default. Please refer to the Druid [JavaScript programming guide](../development/javascript.md) for guidelines about using Druid's JavaScript functionality, including instructions on how to enable it.
 
-### Routing of SQL queries
+### Routing of SQL queries using Strategies
 
-To enable routing of SQL queries, set `druid.router.sql.enable` to `true` (`false` by default). The broker service for a
+To enable routing of SQL queries using strategies, set `druid.router.sql.enable` to `true`. The broker service for a
 given SQL query is resolved using only the provided Router strategies. If not resolved using any of the strategies, the
 Router uses the `defaultBrokerServiceName`. This behavior is slightly different from native queries where the Router
 first tries to resolve the broker service using strategies, then load rules and finally using the `defaultBrokerServiceName`
-if still not resolved.
+if still not resolved. When `druid.router.sql.enable` is set to `false` (default value), the Router uses the
+`defaultBrokerServiceName`.
 
-Routing of native queries is always enabled.
-
-Setting `druid.router.sql.enable` does not affect Avatica JDBC requests. They are routed based on connection ID as
-explained in the next section.
+Setting `druid.router.sql.enable` does not affect either Avatica JDBC requests or Native Queries.
+Native queries are always routed using strategies and load rules as explained above.
+Avatica JDBC requests are always routed based on connection ID as explained in the next section.
```

Comment:
```suggestion
Druid always routes Avatica JDBC requests based on connection ID.
```
Think it's OK not to mention "the next section" since it follows immediately.